internal/printer/storageclass.go

```diff
@@ -0,0 +1,186 @@
+		configFunc:   defaultStorageClassConfig,
+		paramFunc:    defaultStorageClassParameter,
+		storageClass: sc,
+		object:       object,
+	}
+
+	return sch, nil
+}
+
+func (sch *storageClassHandler) Config(options Options) error {
+	out, err := sch.configFunc(sch.storageClass, options)
+	if err != nil {
+		return err
+	}
+	sch.object.RegisterConfig(out)
+	return nil
+}
+
+func (sch *storageClassHandler) Param(options Options) error {
+	if sch.storageClass == nil {
+		return errors.New("can't display parameters for nil storageclass")
+	}
+	sch.object.RegisterItems(ItemDescriptor{
+		Width: component.WidthFull,
+		Func: func() (component.Component, error) {
+			return sch.paramFunc(sch.storageClass, options)
+		},
+	})
+	return nil
+}
+
+func defaultStorageClassConfig(sc *storagev1.StorageClass, options Options) (*component.Summary, error) {
+	return NewStorageClassConfiguration(sc).Create(options)
+}
+
+func defaultStorageClassParameter(sc *storagev1.StorageClass, options Options) (component.Component, error) {
+	return createStorageClassParameterView(sc)
+}
+
+// StorageClassConfiguration is used to create the Storage Class' configuration component
+// when displaying a single Storage Class
+type StorageClassConfiguration struct {
+	storageClass *storagev1.StorageClass
+}
+
+// NewStorageClassConfiguration creates a new StorageClassConfiguration using the supplied Storage Class
+func NewStorageClassConfiguration(sc *storagev1.StorageClass) *StorageClassConfiguration {
+	return &StorageClassConfiguration{
+		storageClass: sc,
+	}
+}
+
+// Create the Configuration Summary component for a Stoage Class
+func (scc *StorageClassConfiguration) Create(options Options) (*component.Summary, error) {
+	if scc.storageClass == nil {
+		return nil, errors.New("Storage Class is nil")
+	}
+	sc := scc.storageClass
+
+	provisioner := sc.Provisioner
+
+	sections := component.SummarySections{}
+	sections.AddText("Provisioner", provisioner)
+
+	if reclaimPolicy := sc.ReclaimPolicy; reclaimPolicy != nil {
+		sections.AddText("Reclaim Policy", string(*reclaimPolicy))
+	}
+
+	if allowVolumeExpansion := sc.AllowVolumeExpansion; allowVolumeExpansion != nil {
+		sections.AddText("Allow Volume Expansion", fmt.Sprintf("%v", *allowVolumeExpansion))
+	}
+
+	if mountOptions := sc.MountOptions; mountOptions != nil {
+		sections.AddText("Mount Options", strings.Join(mountOptions, " "))
+	}
+
+	if volumeBindingMode := sc.VolumeBindingMode; volumeBindingMode != nil {
+		sections.AddText("Volume Binding Mode", string(*volumeBindingMode))
+	}
+
+	summary := component.NewSummary("Configuration", sections...)
+	return summary, nil
+}
+
+func createStorageClassParameterView(sc *storagev1.StorageClass) (component.Component, error) {
+	if sc == nil {
+		return nil, errors.New("Storage Class is nil")
+	}
+
+	columns := component.NewTableCols("Key", "Value")
+	table := component.NewTable("Parameters", "There are no parameters!", columns)
+
+	for key, value := range sc.Parameters {
```

Comment:
`sc.Parameters` is typed `map[string]string` so range does not loop through them deterministically.

This can be fixed via `table.Sort("Key")`